tests/scripts/thread-cert/border_router/test_publish_meshcop_service.py

```diff
@@ -102,11 +102,16 @@ def test(self):
         self.simulator.go(10)
         self.check_meshcop_service(br1, host)
 
+        # check Meshcop-e service
+        self.start_ephemeral_key_mode(br1)
+        self.check_meshcop_e_service(host)
+
         # change ephemeral key mode (ePSKc) and check Meshcop
         br1.ephemeral_key_enabled = False
         self.assertFalse(br1.ephemeral_key_enabled)
         self.simulator.go(10)
         self.check_meshcop_service(br1, host)
+
```

Comment:
We should also check the `meshcop-e` service disappears when the ephemeral key mode expires. At least add a TODO here.

Reply:
accept, modified accordingly. free to have comments for modifications